tests/scripts/validate_response.py

```diff
@@ -5,15 +5,21 @@
 import os
 import sys
 from collections import defaultdict
+from pathlib import Path
 
 from llama_index.embeddings.huggingface import HuggingFaceEmbedding
 from pandas import DataFrame
 from scipy.spatial.distance import cosine, euclidean
 
-sys.path.append(os.path.dirname(os.path.dirname(__file__)))
-from tests.e2e.cluster_utils import create_user, get_user_token, grant_sa_user_access
-from tests.e2e.constants import LLM_REST_API_TIMEOUT
-from tests.e2e.helper_utils import get_http_client
+project_root = Path(__file__).resolve().parent.parent.parent.as_posix()
+sys.path.append(project_root)
```

Comment:
is there actually a reason this code has to be run before the imports?  @asamal4 ?

Reply:
IMHO now it is not needed. It was there because of "cross-package" imports (and it's bit dirty hack ;)